.changeset/clean-mugs-wave.md

```diff
@@ -0,0 +1,26 @@
+---
+"@clerk/clerk-react": minor
+---
+
+Introducing experimental `asProvider`, `asStandalone` and `<X.Outlet/>` for UserButton and OrganizationSwitcher.
+- `asProvider` converts UserButton and OrganizationSwitcher to a provider that defers rendering until `Outlet` is mounted.
+- `Outlet` also accepts `asStandalone` which will skip the trigger of these components and display only the UI of was previously inside the popover. This allows developers to create their own triggers.
```

Comment:
```suggestion
- `<Outlet />` also accepts a `asStandalone` prop. It will skip the trigger of these components and display only the UI which was previously inside the popover. This allows developers to create their own triggers.
```